src/bootstrap/flags.rs

```diff
@@ -161,19 +164,20 @@ impl Flags {
 Usage: x.py <subcommand> [options] [<paths>...]
 
 Subcommands:
-    build, b    Compile either the compiler or libraries
-    check, c    Compile either the compiler or libraries, using cargo check
-    clippy      Run clippy (uses rustup/cargo-installed clippy binary)
-    fix         Run cargo fix
-    fmt         Run rustfmt
-    test, t     Build and run some test suites
-    bench       Build and run some benchmarks
-    doc, d      Build documentation
-    clean       Clean out build directories
-    dist        Build distribution artifacts
-    install     Install distribution artifacts
-    run, r      Run tools contained in this repository
-    setup       Create a config.toml (making it easier to use `x.py` itself)
+    build, b  Compile either the compiler or libraries
+    check, c  Compile either the compiler or libraries, using cargo check
+    clippy    Run clippy (uses rustup/cargo-installed clippy binary)
+    fix       Run cargo fix
+    fmt       Run rustfmt
+    test, t   Build and run some test suites
+    bench     Build and run some benchmarks
+    doc, d    Build documentation
+    clean     Clean out build directories
+    dist      Build distribution artifacts
+    install   Install distribution artifacts
+    run, r    Run tools contained in this repository
+    setup     Create a config.toml (making it easier to use `x.py` itself)
+    suggest   Suggest a subset of tests to run, based on modified files
```

Comment:
it looks like this got autoformatted or something like that; do you mind reverting everything except the new `suggest` line?